core/cibox-project-builder/files/drupal7/scripts/sniffers.yml

```diff
@@ -30,11 +30,12 @@
     build_reports_dir: build_reports
 
     composer_global_require:
-      - drupal/coder:dev-7.x-2.x
-      - squizlabs/php_codesniffer=1.5.6
+      - { package: 'drupal/coder', version: ':dev-8.x-2.x' }
```

Comment:
We can't set coder 8 for Drupal 7 scripts @Krasnyj 
Please use drupal8 folder which is already having Drupal 8 support